tracer/test/test-applications/regression/DogStatsD.RaceCondition/Program.cs

```diff
@@ -18,14 +18,11 @@ static int Main(string[] args)
             {
                 InMemoryLog4NetLogger.Setup();
                 var logger = LogManager.GetLogger(typeof(Program));
-                var ddTraceSettings = TracerSettings.FromDefaultSources();
-                ddTraceSettings.LogsInjectionEnabled = true;
-                ddTraceSettings.TraceEnabled = true;
-                ddTraceSettings.TracerMetricsEnabled = true;
-                GlobalSettings.SetDebugEnabled(true);
-
-                Tracer.Configure(ddTraceSettings);
-                var tracer = Tracer.Instance;
+
+                Environment.SetEnvironmentVariable("DD_TRACE_METRICS_ENABLED", "true");
```

Comment:
Using environment variables won't work here AFAIK (Same reasoning as I gave [in this PR](https://github.com/DataDog/dd-trace-dotnet/pull/3936#discussion_r1144659192)!)